brainscore_core/plugin_management/test_plugin.sh

```diff
@@ -27,22 +27,32 @@ conda install pip
 pip install --upgrade pip setuptools
 
 if [ -f "$CONDA_ENV_PATH" ]; then
-  output=$(conda env update --file $CONDA_ENV_PATH 2>&1)
+  conda env update --file $CONDA_ENV_PATH 2>&1
 fi
 if [ -f "$PLUGIN_SETUP_PATH" ]; then
-  output=$(pip install $PLUGIN_PATH 2>&1)
+  pip install $PLUGIN_PATH 2>&1
 fi
 if [ -f "$PLUGIN_REQUIREMENTS_PATH" ]; then
-  output=$(pip install -r $PLUGIN_REQUIREMENTS_PATH 2>&1)
+  pip install -r $PLUGIN_REQUIREMENTS_PATH 2>&1
 fi
 
 output=$(python -m pip install -e ".[test]" 2>&1) # install library requirements
 
 ### RUN GENERIC TESTING
 if [ "$GENERIC_TEST_PATH" != False ]; then
-  pytest -m "$PYTEST_SETTINGS" "-vv" $GENERIC_TEST_PATH "--plugin_directory" $PLUGIN_PATH "--log-cli-level=INFO" "--junitxml" $XML_FILE
+  if [ "${OPENMIND}" ]; then
+    pip install junitparser
+    PLUGIN_XML_FILE="$PLUGIN_NAME"_"$XML_FILE"
+    pytest -m "$PYTEST_SETTINGS" "-vv" $GENERIC_TEST_PATH "--plugin_directory" $PLUGIN_PATH "--junitxml" $PLUGIN_XML_FILE --capture=no -o log_cli=true;
+    junitparser merge $XML_FILE $PLUGIN_XML_FILE $XML_FILE
```

Comment:
I don't think you need to merge and remove yet here. Looks like this occurs below in the ### RUN TESTING STEP. Also, I think you can leave it as $XML_FILE so the merging process with plugin test results is more understandable